src/ast.js

```diff
@@ -247,8 +247,21 @@ var operatorMap = {};
     operatorMap[TOK.T_PERCENT] = Sk.astnodes.Mod;
 }());
 
+function py3_operators(py3) {
+    if (py3) {
+        operatorMap[TOK.T_AT] = Sk.astnodes.MatMult;
+    } else {
+        if (operatorMap[TOK.T_AT]) {
+            delete operatorMap[TOK.T_AT];
+        }
+    }
+}
+Sk.exportSymbol("Sk.ast.py3_operators", py3_operators);
```

Comment:
Erm...should we really be exporting as `Sk.ast.py3_operators` when the function is not actually set as an attribute on `Sk.ast` object? This seems a bit odd.